pkg/util/net/http.go

```diff
@@ -67,6 +70,9 @@ func SetTransportDefaults(t *http.Transport) *http.Transport {
 	if t.TLSHandshakeTimeout == 0 {
 		t.TLSHandshakeTimeout = defaultTransport.TLSHandshakeTimeout
 	}
+	if err := http2.ConfigureTransport(t); err != nil {
```

Comment:
I don't understand how a custom CA affects or is affected by http2. What am I missing?